.github/services/hdfs/hdfs_cluster/action.yml

```diff
@@ -25,20 +25,10 @@ runs:
       shell: bash
       working-directory: fixtures/hdfs
       run: docker compose -f docker-compose-hdfs-cluster.yml up -d --wait
-    - name: Setup java env
-      uses: actions/setup-java@v4
+    - name: Setup hadoop
+      uses: ./.github/actions/setup-hadoop
       with:
-        distribution: temurin
-        java-version: "11"
-    - name: Setup hadoop env
```

Comment:
Hi, I think adding a separate action only complicates things further. How about using the cache exclusively for `hadoop-3.3.5.tar.gz` here? Here is no other place uses this action.

Reply:
Every action related with hdfs may require this action. They all require setting up Java and a Hadoop environment.